### PR TITLE
Using terms (tokens) in email templates

### PR DESCRIPTION
**Pull request by justinchoi001**
This is a small one in a larger pass for emails in Commerce.

### CODE REVIEW

File: docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md [hunk 1]
```diff
@@ -91,9 +91,4 @@ To change the interval for when Liferay Commerce checks for unsent notifications
 
 ## Additional Notifications
 
-Liferay Commerce offers two snippets to programmatically insert the order creator's name and the order ID.
-
-* \[%ORDER_CREATOR%\] inserts the user who created the order
-* \[%ORDER_ID%\] inserts the Order ID
-
-An extension point lets you implement other notifications: for a shipped order, a restocked product, a carrier exception, or any other event.
+* [Definition of Terms Reference Guide](./definition-of-terms-reference-guide.md)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md:94]: > `* [Definition of Terms Reference Guide](./definition-of-terms-reference-guide.md)`

Broken link.

File: docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Once the Mail settings have been configured, follow these steps to create a noti
     * **From Address**: Your store's email address (for example: _orders@yourstore.com_)
     * **From Name**: Name of the store or a customer service representative
     * **Subject**: Email subject (for example: _Your order has been received._)
-    * **Body**: Email body \*
+    * **Body**: Enter a message. You can create a generic message that uses tokens ("Terms") which contain placeholder values such as customer's name and order ID. (See the [Notification Template Terms Reference Guide](./notification-template-terms-reference-guide.md).)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md:27]: I am very uncertain about calling the tokens, which are placeholders for values that Commerce can fill in, "Terms". Although the product has a "Definition of Terms" - in there, I don't know if those tokens themselves should be referred to as "Terms". I would validate this choice with a developer.

**Comment by jrhoun** [comment]
See my remarks. Thanks!

File: docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md:27]: From https://help.liferay.com/hc/en-us/articles/360032240151-Email-Instance-Settings - this article uses the word "variable"

File: docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/automating-store-emails-by-using-notification-templates.md:94]: fixed.

**Pull request closed without merging**